packages/react/src/lib/fontSizeUtility.ts

```diff
@@ -35,6 +31,9 @@ export const updateCachedRemSize = () => {
  */
 export const pxToRem = (valueInPx: number, baseRemSize?: number): string => {
   if (!baseRemSize && !_documentRemSize) {
+    // there is no way how to reset the cached value, once it's cached, it's cached
```

Comment:
nit-nit: may suggest
- to remove `once it's cached, it's cached`, as it provides no additional value in explanation :)
- rephrase the rest in a more succinct way

> as resetting cached value won't trigger recalculation of site variables, for which originally computed value will stay unchanged